### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by b-buchkevych**
[DEMO LINK](https://b-buchkevych.github.io/layout_product-cards/)
 [TEST REPORT LINK](https://b-buchkevych.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
+      <img 
+        src="images/imac.jpeg"
+        alt="Monoblock photo"
+        class="card__product-photo product-photo"
+      >
+
+      <h2 class="card__product-text text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__product-code product-code">
```
Comment (sTorba24) [inline src/index.html:24]:
```suggestion
      <p class="card__product-code">
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
+        Product code: 195434
+      </p>
+
+      <div class="card__stars-and-review">
```
Comment (sTorba24) [inline src/index.html:28]:
```suggestion
      <div class="card__rate">
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,54 @@
...
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__review review">Reviews: 5</p>
```
Comment (sTorba24) [inline src/index.html:36]:
```suggestion
        <p class="card__review">Reviews: 5</p>
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+.card__product-code {
+  margin: 0 0 16px;
+}
+
+.product-code {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
```
Comment (sTorba24) [inline src/style.css:56]:
```suggestion
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+.card__buy {
+  height: 40px;
+  width: 100%;
+  text-decoration: none;
+  color: #fff;
+  background-color: var(--main-color);
+  border-radius: 5px;
+  font-family: inherit;
```
Comment (sTorba24) [inline src/style.css:132]: it is inherited by default, you should do like that for inputs only

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Gj ! Lets improve it

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,54 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="images/imac.jpeg"
+        alt="Monoblock photo"
+        class="card__product-photo product-photo"
```
Comment (sTorba24) [inline src/index.html:18]: do you need 2 classes here?

Comment (sTorba24) [reply src/index.html:18]: fix everywhere please, i'm sure you do without it

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Well done! Check previous comment, using 2 classes should be fixed everywhere :)

File: src/index.html (hunk 6 above)
Comment (b-buchkevych) [reply src/index.html:18]: In CSS with BEM, the styles responsible for external geometry and positioning are set via the parent block, and the styles of the element itself via a separate class, am I not understanding this correctly?

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
please check your demo links

**Comment by b-buchkevych** [comment]
> please check your demo links

**Pull request closed without merging**